prac_08/convert_miles_km.kv

```diff
@@ -0,0 +1,25 @@
+BoxLayout:
+    orientation: 'vertical'
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput:
+            id: input_miles
+            font_size: 48
+            on_text: app.handle_calculate(self.text)
```

Comment:
Are you using `self.text` to do anything in `handle_calculate`?
You're not required to pass in self.text if you're not using it.